### PR TITLE
Update page title#246

### PR DESCRIPTION
**Pull request by AspectVI**
## Description

Added Titles and Headers to every page by creating the new class Page in the Page.tsx document. This is then used to load the pages instead of 'route'. When pages are loaded this way they are able to have their title and header defined with I the Page call.

Example:
<Page setHeader={setHeader} title="Instruments Page" path="/InstrumentPage" component={InstrumentsPage}/>

## Motivation and Context

This change makes the navigation of the site and the purpose of each page clear by uniquely naming every page based on what it performs. 

Closes Update page title #246

## How Has This Been Tested

Once changes were made a locally run version of the site was used to review the changes and see if anything broke.

## Changes

Changes to The:
DashBoard.tsx
Page.tsx
AppToolbar.tsx

## Tests included/Docs Updated?

- [ ] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: src/components/DashBoard.tsx [hunk 1]
```diff
@@ -23,12 +23,10 @@ import { useCallsData } from 'hooks/call/useCallsData';
 import AppToolbar from './AppToolbar/AppToolbar';
 import CallPage from './call/CallPage';
 import Can, { useCheckAccess } from './common/Can';
-import CreateFeedbackPage from './feedback/CreateFeedbackPage';
-import UpdateFeedbackPage from './feedback/UpdateFeedbackPage';
 import InstitutionPage from './institution/InstitutionPage';
-import MergeInstitutionsPage from './institution/MergeInstitutionPage';
```
Comment (TCMeldrum) [inline src/components/DashBoard.tsx:29]: Looks like you might of removed some pages, might be why some of the tests are failing

File: public/favicon.ico:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (TCMeldrum) [inline public/favicon.ico:Zone.Identifier:1]: I don't think this was meant to be committed

**Review by TCMeldrum — CHANGES_REQUESTED** [review]

File: src/components/DashBoard.tsx (hunk 1 above)
Comment (AspectVI) [reply src/components/DashBoard.tsx:29]: this just complains about not being used then removes its self

Comment (AspectVI) [reply src/components/DashBoard.tsx:29]: when I add it

File: src/components/DashBoard.tsx [hunk 3]
```diff
@@ -51,10 +49,7 @@ import ProposalWorkflowEditor from './settings/proposalWorkflow/ProposalWorkflow
 import ProposalWorkflowsPage from './settings/proposalWorkflow/ProposalWorkflowsPage';
 import UnitTablePage from './settings/unitList/UnitTablePage';
 import ProposalEsiPage from './template/EsiPage';
-import FeedbackTemplatesPage from './template/FeedbackTemplatesPage';
-import GenericTemplatesPage from './template/GenericTemplatesPage';
-import ImportTemplatePage from './template/import/ImportTemplatePage';
-import ProposalTemplatesPage from './template/ProposalTemplatesPage';
```
[the remark this answers is not in the document]
Comment (AspectVI) [reply src/components/DashBoard.tsx:57]: Same with these for It just tells me they are not used and to remove them

File: public/favicon.ico:Zone.Identifier (hunk 2 above)
Comment (TCMeldrum) [reply public/favicon.ico:Zone.Identifier:1]: The whole file sorry

Comment (AspectVI) [reply public/favicon.ico:Zone.Identifier:1]: ok, thanks. Going to keep looking into the other 2 issues.

File: src/components/DashBoard.tsx (hunk 1 above)
Comment (TCMeldrum) [reply src/components/DashBoard.tsx:29]: Looks like when you were developing this feature you were working on an outdated version of the file. So when you merge it into develop it deleted all the new stuff other people added. I think you may need to undo the merge and redo it being careful not to delete the new routes people have added.

Comment (AspectVI) [reply src/components/DashBoard.tsx:29]: Oh. Ok Thanks again tom.

**Comment by AspectVI** [comment]
Closing due to new PR

**Pull request closed without merging**